ui/src/test/java/org/pentaho/di/ui/trans/step/BaseStepDialog_ConnectionLine_Test.java

```diff
@@ -208,6 +208,7 @@ private void test_showDbDialogUnlessCancelledOrValid_ShownOnce( String inputName
                                                                   String expectedResult ) throws Exception {
     DatabaseDialog databaseDialog = mock( DatabaseDialog.class );
     when( databaseDialog.open() ).thenReturn( inputName );
+    when( databaseDialog.getDatabaseMeta() ).thenReturn( createDefaultDatabase() );
```

Comment:
What is the relation of this change and the actual bug being fixed?

Reply:
The fixes to the test don't yield any new tests or test functionality, this just fix problems with the current test that was revealed when I checked in this feature.